docs/ldml/tr35-personNames.md

```diff
@@ -135,6 +135,13 @@ The following features are currently out of scope for Person Names formating:
 
 A draft API for formatting personal names is included in ICU4J 73. (“Draft” means that the full functionality is present, but the API might be refined before it is stabilized.) The implementation can be found at [PersonNameFormatter.java](https://github.com/unicode-org/icu/blob/main/icu4j/main/classes/core/src/com/ibm/icu/text/PersonNameFormatter.java) and [SimplePersonName.java](https://github.com/unicode-org/icu/blob/main/icu4j/main/classes/core/src/com/ibm/icu/text/SimplePersonName.java).
 
+In addition to the settings in this document, it is recommended that implementations provide some additional features in their APIs to allow more control for clients, notably:
+
+1. forceGivenFirst — no matter what the values are in nameOrderLocales or in the NameObject, display the name as givenFirst.
+2. forceSurnameFirst — no matter what the values are in nameOrderLocales or in the NameObject, display the name as surnameFirst.
+3. forceNativeOrdering — no matter what the values are in nameOrderLocales or in the NameObject, display the name with the same ordering as the native locale.
```

Comment:
Not that it matters, but I didn't implement this in ICU.  I think that was partly because what we had at the time wasn't clear on just what this was supposed to do, but also because what I thought it was supposed to do was going to require a lot of extra work in the code that wouldn't normally be worth it.  Is this really worth doing?  If so, you might want to file an ICU ticket.